docker/worker/run.sh.example

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+docker run --rm -it \
+        -e QUEUE_INIT_TIMEOUT='' \
+	-e VALIDATION_TIMEOUT='' \
+	-e VALIDATION_HOME='' \
+	-e VALIDATION_FLAGS= '' \
+        -e S3_VALIDATION_BUCKET='' \
+        -e AWS_BATCH_JOB_ID='' \
+        -e AWS_BATCH_JOB_NODE_INDEX= \
+        -e AWS_ACCESS_KEY_ID='' \
+        -e AWS_SECRET_ACCESS_KEY='' \
+        -e AWS_DEFAULT_REGION='' \
+        --name batch-worker \
```

Comment:
Same as the note in the main/run.sh.  Also, AWS_BATCH_JOB_NODE_INDEX does not have ' ' marks.